packages/builders/scripts/fast-usdc/init-fast-usdc.js

```diff
@@ -38,6 +38,7 @@ const options = {
   },
   chainInfo: { type: 'string' },
   assetInfo: { type: 'string' },
+  noNoble: { type: 'boolean', default: false },
```

Comment:
consider making the option phrased positively. e.g. `connectNoble`. 

I think the default could still be false. connecting to Noble has strictly more dependencies on the environment so makes sense to be opt-in

Reply:
yes, it's more dependencies, but without a connection to noble, the whole notion of "fast usdc" goes away. It's supported only for a limited form of testing. If somebody forgot to set the flag, the normal condition should be that the product works.